torchtext/vocab/vocab_factory.py

```diff
@@ -19,6 +20,7 @@ def vocab(ordered_dict: Dict, min_freq: int = 1,
         min_freq: The minimum frequency needed to include a token in the vocabulary.
```

Comment:
How will `min_freq` interact with `max_tokens`? Does one take precedence over the other? i.e. if `max_tokens` is 50000, but `min_freq` causes the number of tokens to be kept to be 40000, do we ignore one or the other parameter (or give a user warning that both cannot be used at the same time)?

Reply:
I think it makes most sense to apply both.

Reply:
Makes sense! So I guess it's the `min(max_tokens, num_tokens_after_filtering_min_freq)`?